Resources/Prototypes/Catalog/Fills/Crates/fun.yml

```
@@ -597,42 +525,4 @@
-      - id: PlushieGrub
-      - id: PlushieFern
-      - id: PlushieHexadecimal
-      - id: PlushieFlint
-      - id: PlushiePistachio
-      - id: PlushieSeesTheStars
-      - id: PlushieDumbLizard
-      - id: PlushieSqueaks
-      - id: PlushieHaato
-      - id: PlushieTinyLane
-      - id: PlushieChewsTheShoes
+        prob: 0.05
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)